datalad/support/json.py

```diff
@@ -0,0 +1,36 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
```

Comment:
please rename to `json_.py` to not interfer with system wide `json` module.  I was already burnt finding `json.pyc` here